src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -204,6 +256,193 @@ private void displayBySequenceName(Set<ReferenceSequenceTable.TableRow> rows, St
+                            if(pos == previousPos+1 && currentIndel.isDeletion){
+                               currentIndel.ref += ref;
+                            }
+                            else {
+                                if(currentIndel != null) {
+                                    writer.add(currentIndel.getAsVCFRecord());
+                                }
+                                String refPadding = new String(source.queryAndPrefetch(new SimpleInterval(contig, pos-1, pos-1)).getBases());
+                                currentIndel = new MummerIndel(contig, refPadding + ref, refPadding, pos-1, false, true);
+                            }
+                        }
+                        // snp
+                        else {
+                            if(currentIndel != null){
+                                writer.add(currentIndel.getAsVCFRecord());
+                                currentIndel = null;
+                            }
+                            VariantContextBuilder vcfBuilder = new VariantContextBuilder();
+                            VariantContext record = vcfBuilder.chr(contig).start(pos).stop(pos).alleles(ref, alt).make();
+                            writer.add(record);
+                        }
+                        previousPos = pos;
+                    }
+                    if(currentIndel != null){
+                        writer.add(currentIndel.getAsVCFRecord());
+                    }
+            }
+            catch(IOException e){
+                throw new UserException("");
+            }
+            catch(NumberFormatException e){
+                throw new UserException("");
+            }
+        }
+        else{
+            logger.info("No mismatching sequences found.");
+        }
+    }
+
+    private void runFindSNPS(ReferencePair refPair, ReferenceSequenceTable table) {
+        if(refPair.getAnalysis().contains(ReferencePair.Status.DIFFER_IN_SEQUENCE)) {
+            TableColumnCollection columns = new TableColumnCollection(Arrays.asList("Sequence Name", "Position", refPair.getRef1AsString(), refPair.getRef2AsString()));
+            File snpsOutput = new File(baseComparisonOutputDirectory.toPath().toString(), String.format("%s_%s_snps.tsv", refPair.getRef1AsString(), refPair.getRef2AsString()));
+
+            try(FindSNPsOnlyTableWriter writer = new FindSNPsOnlyTableWriter(snpsOutput.toPath(), columns); final ReferenceDataSource source1 = ReferenceDataSource.of(refPair.getRef1().toPath(), true);
+                final ReferenceDataSource source2 = ReferenceDataSource.of(refPair.getRef2().toPath(), true)){
+                writer.writeHeaderIfApplies();
+
+                // find the mismatch sequence
+                for (String sequenceName : table.getAllSequenceNames()) {
+                    Set<ReferenceSequenceTable.TableRow> rows = table.queryBySequenceName(sequenceName);
+                    if (rows.size() == 2) {
```

Comment:
document that rows.size()==2 is equivalent to "there were two md5s found for this name"